features/account/resetting_password_validation.feature

```diff
@@ -0,0 +1,15 @@
+@customer_login
+Feature: Resetting a password validation
+    In order to avoid making mistakes when resetting password
+    As a Visitor
+    I need to be prevented from making mistakes
```

Comment:
`in my address email`? Or something other which specify what kind of mistakes a visitor can do.